scripts/specializations/CpCourseGenerator.lua

```diff
@@ -84,3 +114,33 @@ function CpCourseGenerator:cpDrawFieldPolygon()
         end
     end
 end
+
+function CpCourseGenerator:onReadStream(streamId, connection)
```

Comment:
Not sure about this one. We can generate the boundary server side only, as the custom field border is client side logic. So this should therefore only be event based maybe?

Reply:
No idea :) I added it just in case we need it, if we don't, even better.